core-web/libs/block-editor/src/lib/extensions/components/suggestions/suggestions.component.ts

```diff
@@ -47,15 +47,21 @@ export enum ItemsType {
 export class SuggestionsComponent implements OnInit, AfterViewInit {
     @ViewChild('list', { static: false }) list: SuggestionListComponent;
 
+    // Maybe this should be an @Output() instead of @Input();
     @Input() onSelection: (props: SuggestionsCommandProps) => void;
     @Input() items: DotMenuItem[] = [];
     @Input() title = 'Select a block';
     @Input() noResultsMessage = 'No Results';
     @Input() isOpen = false;
     @Input() currentLanguage = DEFAULT_LANG_ID;
     @Input() allowedContentTypes = '';
+    @Input() showTitle = true;
+    @Input() showURL = false;
+    @Input() loading = false;
+    @Input() waitForSearch = false;
```

Comment:
This is making this component very complex... should we have a new component? Should we add a ng-content approach?